recipes/mss/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set version = "1.1.0" %}
+
+
+package:
+  name: mss
+  version: {{ version }}
+
+source:
+    git_url: https://bitbucket.org/wxmetvis/mss.git
+    git_rev: 1.1.0
+
+build:
+    number: 0
+    script: python setup.py install --record record.txt
+    entry_points:
+      - mss = msui.mss_pyui:main
+
+requirements:
+  build:
+    - basemap ==1.0.7
+    - chameleon >=2.24
+    - curl ==7.26
+    - cycler ==0.10.0
+    - execnet ==1.2.0
```

Comment:
on windows with version pinning  >= I find this pyproj issue. I don't have with basemap-1.0.7.

I check next minutes the server code.

```
  File "C:\Users\user\Miniconda2\envs\mssenv3\Scripts\mss-script.py", line 3, in <module>
    import mslib.msui.mss_pyui
  File "C:\Users\user\Miniconda2\envs\mssenv3\lib\site-packages\mslib\msui\mss_pyui.py", line 59, in <module>
    from mslib.msui import topview
  File "C:\Users\user\Miniconda2\envs\mssenv3\lib\site-packages\mslib\msui\topview.py", line 47, in <module>
    from mslib.msui import ui_topview_window as ui
  File "C:\Users\user\Miniconda2\envs\mssenv3\lib\site-packages\mslib\msui\ui_topview_window.py", line 135, in <module>
    from mpl_qtwidget import MplTopViewWidget
  File "C:\Users\user\Miniconda2\envs\mssenv3\lib\site-packages\mslib\msui\mpl_qtwidget.py", line 42, in <module>
    from mslib.msui import mpl_map
  File "C:\Users\user\Miniconda2\envs\mssenv3\lib\site-packages\mslib\msui\mpl_map.py", line 48, in <module>
    import mpl_toolkits.basemap.pyproj as pyproj
ImportError: No module named pyproj
```

```
# packages in environment at C:\Users\user\Miniconda2\envs\mssenv3:
#
chameleon                 2.24                     py27_0  
basemap                   1.0.8.dev0          np111py27_1    conda-forge
ca-certificates           2016.2.28                     1    conda-forge
curl                      7.48.0                    vc9_2  [vc9]  conda-forge
cycler                    0.10.0                   py27_0    conda-forge
functools32               3.2.3.2                  py27_1    conda-forge
geos                      3.4.2                     vc9_2  [vc9]  conda-forge
hdf4                      4.2.11                    vc9_0  [vc9]  conda-forge
hdf5                      1.8.17                    vc9_2  [vc9]  conda-forge
libnetcdf                 4.4.0                     vc9_1  [vc9]  conda-forge
libpng                    1.6.23                    vc9_0  [vc9]  conda-forge
netcdf4                   1.2.4               np111py27_2    conda-forge
openssl                   1.0.2h                    vc9_0  [vc9]  conda-forge
paste                     2.0.3                    py27_0    conda-forge
pillow                    3.2.0                    py27_0    conda-forge
pyparsing                 2.1.1                    py27_0    conda-forge
pyproj                    1.9.5.1                  py27_0    conda-forge
pyshp                     1.2.3                    py27_0    conda-forge
requests                  2.10.0                   py27_0    conda-forge
setuptools                23.0.0                   py27_0    conda-forge
sip                       4.18                     py27_0    conda-forge
six                       1.10.0                   py27_0    conda-forge
tk                        8.5.19                    vc9_0  [vc9]  conda-forge
zlib                      1.2.8                     vc9_3  [vc9]  conda-forge
execnet                   1.3.0                    py27_1  
freetype                  2.5.5                     vc9_1  [vc9]
jpeg                      8d                        vc9_0  [vc9]
libtiff                   4.0.6                     vc9_2  [vc9]
mss                       1.2.0                         0    local
lxml                      3.6.0                    py27_0  
matplotlib                1.5.1               np111py27_0  
mkl                       11.3.3                        1  
numpy                     1.11.1                   py27_0  
pip                       8.1.2                    py27_0  
pyqt                      4.11.4                   py27_6  
python                    2.7.12                        0  
python-dateutil           2.5.3                    py27_0  
pytz                      2016.4                   py27_0  
qt                        4.8.7                     vc9_8  [vc9]
scipy                     0.17.1              np111py27_1  
vs2008_runtime            9.00.30729.1                  2  
wheel                     0.29.0                   py27_0  

```